packagedoc/additional_docs/Description.rst

```diff
@@ -12,13 +12,452 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 
-The json preprocessor:
+Getting Started
+---------------
 
-.. image:: ./pictures/python3-jsonpreprocessor.png
+The JsonPreprocessor package is a Python3 package which allows developers to handle additional 
+features in json files such as:
 
+* Adds the comments
+* Imports other json files
+* Overwrites existing or add new parameters
+* Uses nested parameter
+* Other features
 
+These json files will be handled by JsonPreprocessor package which returns as result a dictionary 
+object of the deserialized data.
 
-TO BE CONTINUED
+How to install
+--------------
 
+Firstly, clone `python-jsonpreprocessor <https://github.com/test-fullautomation/python-jsonpreprocessor>`_ 
+repository to your machine.
 
+Then go to python-jsonpreprocessor, using the 2 common commands below to build or install this package:
+
+.. code-block::
```

Comment:
Hi Son,
we don't need to write  here the help the build command.
The chapter "how to install" should be like e.g. here: https://github.com/test-fullautomation/robotframework-testresultwebapptool
Thank you,
 Thomas